components/applications/ai/index.md

```diff
@@ -0,0 +1,9 @@
+---
+title: AI Applications
+permalink: /components/applications/ai/
+redirect_from:
+    - /applications/ai/
+
+--- 
+
```

Comment:
I would add a short description of this section.
e.g.

AI applications are used to generate, process, or enrich data by custom models or 3rd Party AI API services. The following AI applications are available:


I would also add a very short description of what the OpenAI app does 
e.g.

- [OpenAI](/components/applications/ai/open-ai/) - Process input data with a custom prompt with OpenAI models

Reply:
@SgtMarmite tagging you